test-main.js

```diff
@@ -12,7 +12,7 @@ System.config({
   defaultJSExtensions: true,
   paths: {
     'angular2/*': 'node_modules/angular2/*.js',
-    'rx': 'node_modules/angular2/node_modules/rx/dist/rx.js'
+    '@reactivex/rxjs/*': 'node_modules/@reactivex/rxjs/*.js',
```

Comment:
I think it should be `node_modules/angular2/node_modules/@reactivex/rxjs/*.js`, because angular2 will install `@reactivex/rxjs` in its own node_modules